packages/common/src/dotlottie-player.ts

```diff
@@ -1596,6 +1596,16 @@ export class DotLottieCommonPlayer {
         if (this.direction === -1) {
           this._container?.dispatchEvent(new Event(PlayerEvents.Complete));
           if (!this.loop) this.setCurrentState(PlayerState.Completed);
+
+          // Fix: First loopComplete is not fired by lottie-web when direction is -1
+          if (this.currentState === PlayerState.Playing && this._loop && this._lottie.playCount === 0) {
+            this._lottie.triggerEvent('loopComplete', {
+              currentLoop: this._lottie.playCount,
+              direction: this.direction,
+              totalLoops: typeof this.loop === 'number' ? this.loop : Infinity,
+              type: 'loopComplete',
+            });
+          }
```

Comment:
if lottie-web fixes this issue from their end, would that make the `loopComplete` fire twice when playing in the reverse direction ?

Reply:
yeah it could fire twice. we may have to revert this change once they fix it.